src/libpriv/rpmostree-scripts.cxx

```diff
@@ -497,9 +495,12 @@ run_script_in_bwrap_container (int rootfs_fd,
           g_strfreev (trace_argv);
         }
 
+      const int script_child_fd = 5;
+      rpmostree_bwrap_take_fd (bwrap, glnx_steal_fd (&script_memfd), script_child_fd);
+      g_autofree char *procpath = g_strdup_printf ("/proc/self/fd/%d", script_child_fd);
       rpmostree_bwrap_append_child_argv (bwrap,
                                          interp,
-                                         postscript_path_container,
+                                         procpath,
```

Comment:
Out of curiosity, does `bwrap` know how to fexecve from a plain FD? Asking because I'm not sure whether SELinux will be happy about exec'ing an entry from /proc.

Reply:
This is actually plain *data* we're passing to an interpreter, almost always `/bin/bash`.